core/src/main/java/org/javarosa/core/model/FormIndex.java

```diff
@@ -438,4 +438,28 @@ public static boolean overlappingLocalIndexesMatch(FormIndex parent, FormIndex c
         }
         return parent.getLocalIndex() == child.getLocalIndex();
     }
+
+    public void assignRefs(FormDef f) {
```

Comment:
Can you mention that this is needed by touchforms so I don't remove it in the future?